CONTRIBUTING.md

```diff
@@ -23,18 +25,6 @@ Use the steps below:
 
    ```
 
-3. On your terminal, type `npm run chat`. Your terminalGPT will start. 😊
+3. On your terminal, type `npm run tgpt -- chat`. Your terminalGPT will start. 😊
 
 <br>
-
-## Extra / Remove from your computer
-
-'npx terminalgpt' doesn't install the terminalgpt package, instead it downloads the package to your pc and directly executes it from the cache.
-
-You can find the package using
-
-`ls ~/.npm/_npx/*/node_modules`
-
-To delete the package, you can use
-
-`rm -r ~/.npm/_npx/*/node_modules/terminalgpt`
```

Comment:
This section doesn't belong in `CONTRIBUTING.md` and is also a duplicate from `README.md`.